src/main/java/edu/internet2/middleware/grouper/changeLog/consumer/Office365ApiClient.java

```diff
@@ -200,6 +218,25 @@ public void addGroup(Group group) {
         }
     }
 
+    protected boolean setupGroupTypeAndSecurityType(Collection<String> groupTypes) {
+        boolean securityEnabled;
+        switch (this.azureGroupType) {
+            case Security:
+                securityEnabled = true;
+                break;
+            case Unified:
+                groupTypes.add("Unified");
+                securityEnabled = false;
+                break;
+            case MailEnabled:
```

Comment:
Perhaps a preference, but I like to see a comment when a case statement falls through to the next condition.